packages/tools/sysutils/jslisten/profile.d/98-jslisten

```diff
@@ -51,7 +51,23 @@ button1=${BTN_BACK60S}
 
 EOF
 		fi
+		if [[ "${HW_DEVICE}" =~ RG351P ]]
```

Comment:
This may be better written as a patch that only applies to the RG351P, or as a function overlay that's located in ~/.config/profile.d.  Ideally we want to minimize the device specific bits in our configs wherever we can.